### DISCUSSION TITLE
A custom text field in a variable product.

### DISCUSSION THREAD

**Discussion by WpEmbraced** [jonas-hoebenreich/wc-donation-platform#35] [Q&A] [answered]
Jonas, thank you for the plugin. It works great. Is there a way to add a text field in a variable product? Something like a simple question where the use can answer by typing in a text field instead of drop-down options and a select box?

Thanks

**Comment by jonas-hoebenreich** [comment, marked as answer]
Hi WpEmbraced,
thanks for reaching out.
This feature is currently not available. But you can use a free plugin like [Flexible Checkout Fields for WooCommerce](https://wordpress.org/plugins/flexible-checkout-fields/) to add extra form fields to your checkout form: https://wcdp.jonh.eu/documentation/integrations/custom-form-fields/
Best regards.

**Reply by WpEmbraced** [reply]
Thank you Jonas. That plugin is helpful too.
